src/com/facebook/buck/jvm/java/coverage/ReportGenerator.java

```diff
@@ -13,6 +13,7 @@
  *******************************************************************************/
 package com.facebook.buck.jvm.java.coverage;
 
+import org.codehaus.plexus.util.FileUtils;
```

Comment:
The report generator is really self contained, it would be nice to be able to keep it that way.